pkgs/applications/misc/input-leap/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, fetchFromGitHub, cmake, curl, xorg, avahi, qtbase, qttools, libei ? null, libportal,
+  mkDerivation, pkg-config, openssl, wrapGAppsHook,
+  avahiWithLibdnssdCompat ? avahi.override { withLibdnssdCompat = true; },
+  fetchpatch
+}:
+
+mkDerivation rec {
+  pname = "input-leap";
+  version = "unstable-2023-05-24";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "5e2f37bf9ec17627ae33558d99f90b7608ace422";
+    hash = "sha256-55RqdRu/Hi2OTiLjAFJ6Gdgg9iO5NIIJCsOkUQjR9hk=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    curl qtbase avahiWithLibdnssdCompat
+  ] ++ (with xorg; [
+    libX11 libXext libXtst libXinerama libXrandr libXdmcp libICE libSM
+  ]) ++ lib.optionals (libei != null) [
+    libei libportal
+  ];
+  nativeBuildInputs = [ pkg-config cmake wrapGAppsHook qttools ];
+
+  cmakeFlags = [
+    "-DINPUTLEAP_REVISION=${builtins.substring 0 8 src.rev}"
+  ] ++ lib.optional (libei != null) "-DINPUTLEAP_BUILD_LIBEI=ON";
+
+  dontWrapGApps = true;
+
+  # Arguments to be passed to `makeWrapper`, only used by qt5’s mkDerivation
+  preFixup = ''
+    qtWrapperArgs+=(
+      "''${gappsWrapperArgs[@]}"
+        --prefix PATH : "${lib.makeBinPath [ openssl ]}"
+    )
+  '';
+  postFixup = ''
+    substituteInPlace "$out/share/applications/input-leap.desktop" --replace "Exec=input-leap" "Exec=$out/bin/input-leap"
+  '';
+
+  meta = {
+    description = "Open-source KVM software";
+    longDescription = ''
+      Input Leap is software that mimics the functionality of a KVM switch.
+    '';
```

Comment:
```suggestion
```

Or at least use the full paragraph.

```
Input Leap is software that mimics the functionality of a KVM switch, which historically 
would allow you to use a single keyboard and mouse to control multiple computers by
physically turning a dial on the box to switch the machine you're controlling at any 
given moment. Input Leap does this in software, allowing you to tell it which machine 
to control by moving your mouse to the edge of the screen, or by using a keypress 
to switch focus to a different system.
```

Reply:
Why do you prefer removing it — is having a `longDescription` discouraged now?

Reply:
`longDescription` is optional

A __good__ `longDescription` is never discouraged.

A __bad__ `longDescription` is discouraged.